src/node/rpc/memberfrontend.h

```diff
@@ -395,10 +395,25 @@ namespace ccf
         if (verify_result != QuoteVerificationResult::VERIFIED)
           return QuoteVerifier::quote_verification_error_to_json(verify_result);
 #endif
+        auto nodes_view = args.tx.get_view(this->network.nodes);
+        bool is_duplicate = false;
+        // TODO(#api): foreach should check the callback's value and be able to
```

Comment:
There's an open issue for this: #60
We should start listing in the issue all the places (like this) where we want this behaviour.

Reply:
I'll handle #60 today